packages/cli/src/ceramic-daemon.ts

```diff
@@ -11,7 +11,7 @@ const DEFAULT_PORT = 7007
 const DEFAULT_NETWORK = 'testnet-clay'
 const toApiPath = (ending: string): string => '/api/v0' + ending
 
-const DEFAULT_ANCHOR_SERVICE_URL = "https://cas.3box.io:8081"
+const DEFAULT_ANCHOR_SERVICE_URL = "https://cas-clay.3box.io:8081"
```

Comment:
should be `https://cas-clay.3boxlabs.com`

Reply:
Done